tests/drivers/build_all/i2c.dtsi

```diff
@@ -517,3 +517,9 @@ test_i2c_bq274xx: bq27xx@47 {
 	taper-current = <45>;
 	terminate-voltage = <3000>;
 };
+
+test_i2c_dps310: dps310@48 {
+		compatible = "infineon,dps310";
+		label = "DPS310";
+		reg = <0x48>;
+};
```

Comment:
move this change to the previous commit, so it goes with `tests/drivers/build_all/sensors_a_h.conf`